src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -687,6 +684,55 @@ public AnnotationModel AddAnnotation(string text, Guid id)
         }
 
         /// <summary>
+        /// Sets the event on Annotaiton model.
+        /// When a node is removed from a group, this event will be
+        /// used to include that node again in that group (UNDO operation)
+        /// </summary>
+        /// <param name="model">The model.</param>
+        internal void SetGetModelsForGrouping(AnnotationModel model)
```

Comment:
Let's talk about this, I don't really understand what this method is doing.  `SetGet`?